### ISSUE TITLE
No apps associated with package gfont or its dependencies

### ISSUE THREAD

**Issue by 34j** [nureon22/gfont#4]
### Describe the bug

```shell
pipx install "git+https://github.com/nureon22/gfont.git"

No apps associated with package gfont or its dependencies. If you are attempting to install a library, pipx should not
be used. Consider using pip or a similar tool instead.
```

### About your system

operation system :

python version :

gfont version :

**Comment by nureon22** [comment]
You need to build the wheel file first to install the package. Please See https://github.com/nureon22/gfont?tab=readme-ov-file#installation.

**Comment by 34j** [comment]
It works now, thank you

**Comment by nureon22** [comment]
@34j This fixed in commit [53aa43e](https://github.com/nureon22/gfont/commit/53aa43eabf55486bef28faba0218ac8de214bf41).
